package.json

```diff
@@ -60,18 +60,18 @@
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
-    "@babel/cli": "^7.22.5",
-    "@babel/core": "^7.22.5",
-    "@babel/eslint-parser": "^7.22.5",
-    "@babel/node": "^7.22.5",
+    "@babel/cli": "^7.22.9",
+    "@babel/core": "^7.22.9",
+    "@babel/eslint-parser": "^7.22.9",
+    "@babel/node": "^7.22.6",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-do-expressions": "^7.22.5",
     "@babel/plugin-proposal-export-default-from": "^7.22.5",
     "@babel/plugin-proposal-logical-assignment-operators": "^7.20.7",
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@babel/plugin-proposal-pipeline-operator": "^7.22.5",
-    "@babel/preset-env": "^7.22.5",
+    "@babel/preset-env": "^7.22.9",
     "@babel/preset-react": "^7.22.5",
     "@storybook/addon-a11y": "7.0.20",
```

Comment:
I'll look into it!

Reply:
There is an issue with the latest storybook version (just released this morning) and its compatibility with yarn v1. There is a [potential workaround](https://github.com/storybookjs/storybook/issues/22431#issuecomment-1630086092), but based on offline discussion we will wait until the next grommet post-release dependency upgrade to revisit since this release is so new. In the past, storybook has had unreliable releases and released patch fixes later on.